### DISCUSSION TITLE
ReferenceError: fetch is not defined when using fromFetch of rx.js

### DISCUSSION THREAD

**Discussion by ClickHere0521** [ReactiveX/rxjs#7024] [Q&A] [answered]
I am very early to rxjs so my question may be a little stupid for you. but if you help me that'd be really great for me.
I use fromFetch of rxjs but got an error like `ReferenceError: fetch is not defined`
Do I need to install `node-fetch` or any other libriries? Is it possible to use `fromFetch` without any library?

`import { fromFetch } from 'rxjs/fetch';`

**Comment by kwonoj** [comment, marked as answer]
RxJS does not polyfill runtime environment api surfaces. It is up to user / application to provide those.

**Reply by ClickHere0521** [reply]
Thanks @kwonoj 
So in node.js environment, it's impossible to use `fromFetch`. is that right?
